vicodyn/include/cocaine/vicodyn/peer.hpp

```diff
@@ -95,19 +95,19 @@ class peers_t {
         std::unique_ptr<metrics::usts::ewma<clock_t>> timings_ewma_;
 
     public:
-        app_service_t(clock_t::duration timings_window);
+        app_service_t(std::chrono::milliseconds timings_window);
 
         auto ban(std::chrono::milliseconds timeout) -> void;
-
         auto banned() const -> bool;
+        auto banned_for() const -> std::chrono::nanoseconds;
```

Comment:
Could it return `std::chrono::microseconds`? Currently it is used only in combination with `std::chrono::duration_cast<std::chrono::milliseconds>()`.

Reply:
Why? We don't do extra `dynamic_cast<>` and `nanoseconds` is better for interface because more precise.